ingest_api/runtime/src/auth.py

```diff
@@ -53,8 +55,10 @@ def validated_token(
     return token
 
 
-def get_username(token: Annotated[Dict[Any, Any], Depends(validated_token)]):
-    return token["username"]
+def get_username(token: Annotated[Dict[Any, Any], Depends(validated_token)]) -> str:
+    logger.info(f"\nToken {token}")
+    result = token["username"] if "username" in token else token.get("sub", None)
```

Comment:
I'm mixed about having a default when we might want to fail but because this depends on validation I think it is OK? We can decide after it works or not

Reply:
https://github.com/NASA-IMPACT/veda-backend/pull/365/commits/dbb62fbc15ce309ef5d12abe8bbee600c66a48ac Perhaps we can raise the error instead?

Reply:
[f640fdf](https://github.com/NASA-IMPACT/veda-backend/pull/365/commits/f640fdf537e8d56cfd4e8321be0e25e97e029df1) change reverted since we don't want to be blocked and could forget about this change later on